lnrpc/routerrpc/router.proto

```diff
@@ -85,6 +89,121 @@ message RouteFeeResponse {
     int64 time_lock_delay = 2;
 }
 
+message SendToRouteRequest {
+    /// The payment hash to use for the HTLC.
+    bytes payment_hash = 1;
+
+    /// Route that should be used to attempt to complete the payment.
+    lnrpc.Route route = 2;
+}
+
+message SendToRouteResponse {
+    /// The preimage obtained by making the payment.
+    bytes preimage = 1;
+
+    /// The failure message in case the payment failed.
+    Failure failure = 2;
+}
+
+message Failure {
+    enum FailureCode {
+        RESERVED = 0;
```

Comment:
Why reserved?

Reply:
`routerrpc/router.proto: The first enum value must be zero in proto3.`

I wanted to keep the numbers equal to the lightning spec.

Reply:
Add this as a comment?

Reply:
Added.